config/initializers/paper_trail.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2022, Schweizer Wanderwege. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+PaperTrail::Version.class_eval do
+  def perpetrator
+    if whodunnit.present? && whodunnit_type.present?
+      whodunnit_type.constantize.find(whodunnit.to_i) 
```

Comment:
Das wird auf die Schnauze fallen sobald ein Service Token oder eine Person in der zwischenzeit hart gelöscht wurde.

Reply:
true && thx && fixed